Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -422,7 +422,7 @@ def main():
     params = demisto.params()
     Client.severity = params.get('severity', '').upper()
     Client.query = params.get('query', Client.query)
-    Client.tag = params.get('tag', Client.tag)
+    Client.tag = params.get('tag', Client.tag) if params.get('tag', Client.tag) else params.get('feedTags', Client.tag)
```

Comment:
since `Client.tag` is not falsy, this will result in always using `tag` and not `feedTags`.

image the next scenario:
1) setting up the integration (`tags` is set)
2) upgrading pack / content
3) switching tags value (now `feedTags` is set)
in the current state, step 3 will not have any effect.

Reply:
I changed it to what @esharf  suggested, this will have an effect in my understanding.